src/sagemaker/model_monitor/model_monitoring.py

```diff
@@ -1666,12 +1676,19 @@ def suggest_baseline(
                 Only meaningful when wait is True (default: True).
             job_name (str): Processing job name. If not specified, the processor generates
                 a default job name, based on the image name and current timestamp.
+            monitoring_config_override (DataQualityMonitoringConfig): monitoring_config object to
+                override the global monitoring_config parameter of constraints suggested by
+                Model Monitor Container. If not specified, the default values of the parameters
+                is suggested.
```

Comment:
Update this comment to reflect that "If not specified, the method suggested by container is not overridden or values suggested by container is set". Lets not call out that "default values of parameters is suggested"

Reply:
Done.